shared/api-spec/representative-points-response.json

```diff
@@ -10,16 +10,9 @@
             "type": "string"
           },
           "values": {
-            "items": [{
-                "type": "number"
-              },
-              {
-                "type": "number"
-              },
-              {
-                "type": "number"
-              }
-            ],
+            "items": {
```

Comment:
This thing went from being an array to a single dict?

Reply:
Discussed IRL